src/features/defi/providers/fox-eth-lp/hooks/useFoxEthLiquidityPool.ts

```diff
@@ -50,6 +54,22 @@ export const useFoxEthLiquidityPool = (accountAddress: string | null) => {
   const ethAsset = useAppSelector(state => selectAssetById(state, ethAssetId))
   const foxAsset = useAppSelector(state => selectAssetById(state, foxAssetId))
   const lpAsset = useAppSelector(state => selectAssetById(state, foxEthLpAssetId))
+
+  const accountId = useMemo(
+    () =>
+      accountAddress
+        ? toAccountId({
+            chainId: ethChainId,
+            account: accountAddress,
+          })
+        : '',
```

Comment:
The choice of ternary as a syntax might be boilerplaty but is on purpose, nullish coalescing operator won't do the trick since `null` won't be treated as an `AccountId`ish (effectively a flavored string), but `''` will